servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/OffloadAllExecutionStrategy.java

```diff
@@ -31,23 +31,23 @@ private OffloadAllExecutionStrategy() {
     }
 
     @Override
-    public <T> Single<T> offloadSend(final Executor fallback, final Single<T> original) {
-        return original.subscribeOn(fallback);
+    public <T> Single<T> offloadSend(final Executor executor, final Single<T> original) {
```

Comment:
What equivalent class is used for non-test code?

Reply:
You introduced `OffloadFromIOExecutionStrategy` instead of `OffloadAllExecutionStrategy`. As the result, `OffloadAllExecutionStrategy` is not used by any production code anymore.

Reply:
Moved `OffloadAllExecutionStrategy` to test sources.